include/bluetooth/mesh/cfg_cli.h

```diff
@@ -183,6 +183,36 @@ int bt_mesh_cfg_gatt_proxy_set(uint16_t net_idx, uint16_t addr, uint8_t val,
 int bt_mesh_cfg_net_transmit_get(uint16_t net_idx, uint16_t addr,
 			  uint8_t *transmit);
 
+/** @brief Get the target node's identity state.
+ *
+ *  @param net_idx     Network index to encrypt with.
+ *  @param addr        Target node address.
+ *  @param key_net_idx Network key index.
+ *  @param status      Status response parameter.
+ *  @param identity    Identity state.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_cfg_node_identity_get(uint16_t net_idx, uint16_t addr,
+				  uint16_t key_net_idx, uint8_t *status, uint8_t *identity);
+
+/** @brief Set the target node's identity state.
+ *
+ *  @param net_idx     Network index to encrypt with.
+ *  @param addr        Target node address.
+ *  @param key_net_idx Network key index.
+ *  @param val         New Node Identity state, should be one of
+ *                     @ref BT_MESH_NODE_IDENTITY_STOPPED or
+ *                     @ref BT_MESH_NODE_IDENTITY_RUNNING.
+ *  @param status      Status response parameter.
+ *  @param identity    Identity state.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_cfg_node_identity_set(uint16_t net_idx, uint16_t addr,
+				  uint16_t key_net_idx, uint8_t val,
+				  uint8_t *status, uint8_t *identity);
```

Comment:
I don't think we have the habit of returning the "current value" parameter for any other "set" client APIs, so I don't think we should do it here either, i.e. you should remove the last parameter.